src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const bigReduction = 50;
+  const smallReduction = 20;
+  const price = 40;
+  let result = days * price;
```

Comment:
Since `result` is not reassigned after its initial assignment, it can be declared with `const` instead of `let` to prevent accidental changes.